Python/sort/README.md

```diff
@@ -1 +1,24 @@
-a
+ ## Sorting Algorithms
+
+In computer science, a sorting algorithm is an algorithm that puts elements of a list in a certain order. The most frequently used orders are numerical order and lexicographical order. Efficient sorting is important for optimizing the efficiency of other algorithms that require input data to be in sorted lists. 
```

Comment:
This part is plagiarised from the wiki, you can either mention the source or can write a definition of your own.

Reply:
Oh yes, I completely forgot to mention the source.
Updating it!